atc/db/build.go

```diff
@@ -452,7 +453,7 @@ func (b *build) Delete() (bool, error) {
 // build was aborted before it was started.
 func (b *build) MarkAsAborted() error {
 	_, err := psql.Update("builds").
-		Set("status", string(BuildStatusAborted)).
+		Set("aborted", true).
```

Comment:
If we're no longer setting a build's status to `BuildStatusAborted`, it may make sense to remove that type entirely and pull on that thread. There's a few api tests which mock out the build object and assume returning a status of `BuildStatusAborted`  is still what indicates the aborted-ness of a build.